rllib/algorithms/apex_dqn/apex_dqn.py

```diff
@@ -657,10 +668,8 @@ def on_worker_failures(
             self._sampling_actor_manager.add_workers(new_workers)
 
     @override(Algorithm)
-    def _compile_iteration_results(self, *, step_ctx, iteration_results=None):
-        result = super()._compile_iteration_results(
-            step_ctx=step_ctx, iteration_results=iteration_results
-        )
+    def _compile_iteration_results(self, *args, **kwargs):
```

Comment:
curious, why change func arguments to this generic types?

Reply:
To make things less prone to forward compat errors in the future.
If the args are not really needed in the child implementation, then why not? :)

Reply:
I see. a minor suggestion may be to name the arguments *unused_args and **unused_kwargs to make the intention clearer.
if in the future we do need some parameters, it's better to write it out.